content/pt-br/docs/concepts/security/windows-security.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Segurança para Nós Windows
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+Esta página descreve considerações de segurança e boas práticas específicas para o sistema operacional Windows.
+
+<!-- body -->
+
+## Proteção para dados Secret nos Nós
+
+No Windows, os dados do Secret são escritos em texto não-encriptado no Nó local do
+armazenamento (em comparação ao uso de tmpfs / sistemas de arquivo em memória no Linux). Como um operador do cluster, você deve tomar as duas medidas adicionais a seguir:
+
+1. Use ACLs em arquivos para proteger a localização do arquivo Secrets.
+2. Aplicar criptografia a nível de volume usando
```

Comment:
nit: I think you could use "Aplique", since you used imperative tense in item 1.

Reply:
Done!

Reply:
Ok!